CommunityFridgeMapApi/functions/fridges/v1/app.py

```diff
@@ -0,0 +1,54 @@
+import os
+import logging
+import json
+
+try:
+    from db import get_ddb_connection, Fridge
+except:
+    # If it gets here it's because we are performing a unit test. It's a common error when using lambda layers
+    # Here is an example of someone having a similar issue:
+    # https://stackoverflow.com/questions/69592094/pytest-failing-in-aws-sam-project-due-to-modulenotfounderror
+    from dependencies.python.db import get_ddb_connection, Fridge
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class FridgHandler:
```

Comment:
#### Nit
Typo here: Fridg"e"Handler.

Reply:
whoops 😅, thanks for catching that!

Reply:
done 74b226e8cb66bb72f9d30629eb3fe43f1f4bbf2a